lighthouse-core/gather/driver.js

```diff
@@ -96,6 +96,29 @@ class Driver {
     this.evaluate = this.executionContext.evaluate.bind(this.executionContext);
     /** @private @deprecated Only available for plugin backcompat. */
     this.evaluateAsync = this.executionContext.evaluateAsync.bind(this.executionContext);
+
+    // A shim for sufficient coverage of targetManager functionality.
```

Comment:
more comment, maybe:

`// Legacy driver does not need target management because the legacy driver explicitly handles Target.attachedToTarget events. See _handleTargetAttached`